bitcoin/examples/create-p2wpkh-address.rs

```diff
@@ -0,0 +1,27 @@
+extern crate bitcoin;
+
+use bitcoin::{Address, CompressedPublicKey, PrivateKey, Network};
+use bitcoin::secp256k1::{rand, Secp256k1};
+
+// Generate a P2WPKH (pay-to-witness-public-key-hash) address and print it
+// along with the associated private key needed to transact.
+
+fn main() {
+    // Create a new secp256k1 elliptic curve.
+    let s = Secp256k1::new();
```

Comment:
I'd prefer not to use `s`. I personally love single character local varibales but this repo tends to oppose them. In this instance `secp` is better IMO.

Reply:
I'm impartial but since you ask I'll change it.